src/pheval/analyse/benchmark_generator.py

```diff
@@ -1,19 +1,18 @@
-from collections import defaultdict
 from dataclasses import dataclass
 from typing import Callable
 
 from pheval.analyse.benchmarking_data import BenchmarkRunResults
 from pheval.analyse.disease_prioritisation_analysis import benchmark_disease_prioritisation
 from pheval.analyse.gene_prioritisation_analysis import benchmark_gene_prioritisation
-from pheval.analyse.run_data_parser import TrackInputOutputDirectories
+from pheval.analyse.run_data_parser import RunConfig, SinglePlotCustomisation
 from pheval.analyse.variant_prioritisation_analysis import benchmark_variant_prioritisation
 from pheval.constants import (
-    DISEASE_PLOT_FILE_PREFIX,
     DISEASE_PLOT_Y_LABEL,
-    GENE_PLOT_FILE_PREFIX,
+    DISEASE_PRIORITISATION_TYPE_STR,
     GENE_PLOT_Y_LABEL,
-    VARIANT_PLOT_FILE_PREFIX,
+    GENE_PRIORITISATION_TYPE_STR,
     VARIANT_PLOT_Y_LABEL,
+    VARIANT_PRIORITISATION_TYPE_STR,
 )
```

Comment:
Why define these in constants.py? They are only used here, so define them here and cut down on the import cruft and keep the values local to their usage..